src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1229,23 +1229,125 @@ private int SkipBlockComment(int i)
             return i;
         }
 
-        private static char Backtick(char c)
+        private string Backtick(char c)
```

Comment:
BTW `'\u'` does not compile in C#. It generates the following error:
![image](https://user-images.githubusercontent.com/5177512/26941464-cc727300-4c3b-11e7-8263-d85502e0ed6e.png).
Now Backtick could default to just returning `u` and we could test for that in each of the 6 call sites but we're no longer testing for `\u`.  This is starting to feel a bit less elegant.